src/renderers/common/Renderer.js

```diff
@@ -1075,9 +1075,15 @@ class Renderer {
 
 			this._currentRenderBundle = renderBundle;
 
-			const opaqueObjects = renderList.opaque;
+			const {
+				lightsNode,
```

Comment:
If you extract `lightsNode` from the render list, we can remove it from the method signature.